src/modules/assetus/briefs4assetus/asset_brief.go

```diff
@@ -103,12 +104,13 @@ func (v *AssetBrief) Validate() error {
 		return validation.NewErrBadRecordFieldValue("category", "unknown asset category: "+string(v.Category))
 	}
 
-	if strings.TrimSpace(v.Make) == "" {
-		return validation.NewErrRecordIsMissingRequiredField("make")
-	}
-	if strings.TrimSpace(v.Model) == "" {
-		return validation.NewErrRecordIsMissingRequiredField("model")
-	}
+	// if strings.TrimSpace(v.Make) == "" {
```

Comment:
This need to be inside IF block for vehicle only

Reply:
```
case const4assetus.AssetCategoryVehicle:
		if strings.TrimSpace(v.Make) == "" {
			return validation.NewErrRecordIsMissingRequiredField("make")
		}
		if strings.TrimSpace(v.Model) == "" {
			return validation.NewErrRecordIsMissingRequiredField("model")
		}
		if err := checkType(const4assetus.AssetVehicleTypes); err != nil {
			return err
		}
```
I moved Make and Model validation here.